### PR TITLE
[Cog Service AnomalyDetector] add more supported granularity

### PR DESCRIPTION
**Pull request by moreOver0**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by moreOver0** [comment]
Can anyone help check why the C# sdk pr didn't get triggered automatically?

**Comment by moreOver0** [comment]
@yangyuan Can you help?

**Review by yangyuan — APPROVED** [review]

**Comment by moreOver0** [comment]
@tjprescott Can you help look into why the pr for csharp sdk doesn't get triggered automatically?

**Review by tjprescott — APPROVED** [review]

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 1]
```diff
@@ -200,7 +200,8 @@
         "weekly",
         "daily",
         "hourly",
-        "minutely"
+        "minutely",
+        "secondly"
```
Comment (tjprescott) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json:204]: This isn't pertinent to the swagger review, but just feedback for your service.

The term "minutely" is not common--I've never heard it used an had to look it up to verify it's actually a word in the dictionary (surprised to find that it is). However, the term "secondly" does NOT mean what is implied here (i.e. "per second"). It means "In second place". I would recommend deprecating these enum values with replacing them with either pattern:
`week, day, hour, minute, second` or `perWeek`, `perDay`, `perHour`, `perMinute`, `perSecond`. I would recommend the first. It's simpler and consistent with what I've seen in other services like Azure Monitor.

**Review by tjprescott — APPROVED** [review]
Approved, but please do look at my comment on the use of the terms "minutely" and "secondly".

**Pull request merged**